SLSA v1: re-add exceptions for tenant-generated provenance fields (#779)
slsa-framework/slsa: docs/provenance/v1.md

-   The scope of what this ID represents.
-   The claimed SLSA Build level.
-   The accuracy and completeness guarantees of the fields in the provenance.

> -   Any fields that are generated by the tenant-controlled build process and not
>     verified by the trusted control plane, aside from the `subject`.

Reviewer: Nit: I find this phrasing clearer and more in-keeping with the rest of the language in the spec
```suggestion
-   Any fields that are generated by the tenant-controlled build process and not
    verified by the trusted control plane, except for the `subject`.
```
